static/js/components/CourseList.js

```diff
@@ -1,19 +1,50 @@
 import React from 'react';
 
+import { makeCourseStatusDisplay } from '../util/util';
+
 class CourseList extends React.Component {
   render() {
-    const { courseList } = this.props;
+    const { dashboard, courseList } = this.props;
+    let dashboardLookup = {};
+    for (let course of dashboard.courses) {
+      dashboardLookup[course.id] = course;
+    }
+
+    let tables = courseList.programList.map(program => {
```

Comment:
is the `programList` here the same as the `programList` that's dispatched to the store after we use the `fetchCourseList` method in `actions/index.js`?